src/Docker.jl

```diff
@@ -93,7 +94,17 @@ function build_docker_image(root_path::String, uid::Cint, gid::Cint; verbose::Bo
         open(`docker import - $(image_name)`, "w", verbose ? stdout : devnull) do io
             # we need to record permisions, so can't use Tar.jl
             cd(root_path) do
-                run(pipeline(`tar -c --owner=$(uid) --group=$(gid) .`, stdout=io))
+                get_cmd = tar -> `$(tar) -c --owner=$(uid) --group=$(gid) .`
+                if Sys.iswindows() # Tar_jll does not currently support Windows
```

Comment:
Instead of hardcoding "Tar_jll doesn't work on Windows", it's better to check `Tar_jll.is_available()`

Reply:
Oh nice!